package.json

```diff
@@ -40,31 +45,46 @@
     "typeface-roboto": "^1.0.0"
   },
   "devDependencies": {
-    "@neutrinojs/airbnb": "9.0.0",
-    "@neutrinojs/jest": "9.0.0",
-    "@neutrinojs/react": "9.0.0",
+    "@babel/core": "^7.26.0",
+    "@babel/eslint-parser": "^7.25.9",
+    "@babel/plugin-transform-runtime": "^7.25.9",
+    "@babel/preset-env": "^7.26.0",
+    "@babel/preset-react": "^7.26.3",
+    "@babel/runtime": "^7.26.0",
     "@pollyjs/adapter-fetch": "^5.0.0",
     "@pollyjs/adapter-node-http": "^5.0.0",
     "@pollyjs/core": "^5.0.0",
     "@pollyjs/persister-fs": "^5.0.0",
-    "@testing-library/react": "10.0.6",
+    "@testing-library/react": "^16.1.0",
```

Comment:
Note that you need to call `render` with `legacyRoot` with this version:

https://testing-library.com/docs/react-testing-library/api#legacyroot

Otherwise you get an error in React < 18.